contrib/gitian/README.md

```diff
@@ -83,7 +87,7 @@ Docker
 Prepare for building with docker:
 
 ```bash
-sudo apt-get install git make curl docker.io
+sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get install make autoconf g++ libtool pkg-config gperf cmake python3-zmq libdbus-1-dev libharfbuzz-dev git curl docker.io
```

Comment:
Excessive use of sudo. Should combine all the commands into one sudo invocation, or just use su, or sudo bash.

None of these extra packages are needed on the host system. They're only needed inside the docker container, and the docker build scripts already pull them down.

Reply:
Yep, these are the dependencies for if you want to build with depends.

Reply:
> Excessive use of sudo. Should combine all the commands into one sudo invocation, or just use su, or sudo bash.
 
Will try.

> None of these extra packages are needed on the host system. They're only needed inside the docker container, and the docker build scripts already pull them down.

I tried twice with the previous set of packages and got the same error - missing tooling during Android compilation. But I agree, that I should minimize the list better.

Reply:
But this would mean, that the Dockerfile has failed to download the Android's dependencies. I will look there.

Reply:
The issue hasn't repeated anymore after I simply made apt update and upgrade.